app/models/patient.rb

```diff
@@ -140,12 +140,13 @@ def latest_transfer
 
   # Patients who are eligible for reminders
   scope :reminder_eligible, lambda {
-    where(purged: false)
+    monitoring_open
```

Comment:
Okay, I definitely see that issue. I think that we will want to wait to incorporate `monitoring_open` until we also incorporate `.or(has_eligible_dependents)` at the same time.

This is because the way the `better_reminder_eligible` scope is set up is that that everything except the `.or(has_eligible_dependents)` is to check if the patient itself is eligible and the `.or(has_eligible_dependents)` is kind of a back up check to see if that patient has any eligible dependents (even if they were not eligible themself).

Better Reminder Eligible Snippet:
```
scope :better_reminder_eligible, lambda {
  joins(:dependents)
    .monitoring_open
    .within_preferred_contact_time
    .has_not_reported_recently
    .is_being_monitored
    .has_usable_preferred_contact_method
    .where('patients.id = patients.responder_id')
    .where(pause_notifications: false)
    .reminder_not_sent_recently
    .or(has_eligible_dependents)
}
```

Reply:
Yep exactly. So to make sure we're on the same page, you're saying you'd rather revert the `monitoring_open` change here and include a `.or(has_eligible_dependents)` in a subsequent branch for 1250, right?

Reply:
Yeah, I will do that today. Should I add another criteria in or are you thinking we should only incorporate `has_usable_preferred_contact_method` this time around?

Reply:
Swapping in `.has_usable_preferred_contact_method` here seems like an easy, harmless addition you could throw in now. Although I don't think it will make any difference in performance, as it's doing the exact same thing. https://github.com/SaraAlert/SaraAlert/blob/f3eb292f641ac4584d4060a236e925d512de005e/app/models/patient.rb#L146

Speaking of which, as I attempt to profile the performance of this before and after, have you done any profiling with this change? Do we already have any automated performance tests for it? Do you have a sense of if/how much this helps performance?

Reply:
Once we merge in https://github.com/SaraAlert/SaraAlert/pull/731 it will be much easier to gauge the performance increases in an automated fashion. That has a micro-benchmark for the performance of the scope itself, as well as a benchmark for running the `send_assessments_job`.

 I don't expect that one change to have any performance increase because of the reasons you stated - just a readability improvement. Because of that I was suggesting that we could incorporate another helper scope. We could add something like `within_preferred_contact_time` or even `.or(has_eligible_dependents)`

Reply:
As I think about it more, it seems like we might want to incorporate `.or(has_eligible_dependents)` first so that as we incorporate more constraints to the part of the scope that checks if the HoH/patient not in household is eligible we do not accidentally exclude patients who are not eligible themselves but have eligible dependents.

Reply:
Point taken, although I'm also hesitant to throw in `.or(has_eligible_dependents)`, because there's a lot of conditions in that scope. And as far as I can tell, those are filter conditions we just weren't applying previously. So to me that seems like not only a performance change but it would change which people are actually receiving assessments. Are we currently sending assessments to HoHs _without_ eligible dependents?

Reply:
@holmesie might be better able to weigh in on that question.

I'm okay if we're not ready to incorporate `.or(has_eligible_dependents)` yet - I recognize that it has a lot of criteria. I think a safer alternative will be to incorporate both `has_usable_preferred_contact_method` (for readabilty change) and `within_preferred_contact_time` (for performance bump) for this time around. 

By adding `within_preferred_contact_time` we can also remove the entire `unless send_now ...` block and `send_now` argument from `Patient#send_assessment` since the timing criteria would now be checked on the DB side.